app/src/main/java/de/tum/in/tumcampusapp/activities/PersonsSearchActivity.java

```diff
@@ -74,6 +58,20 @@ public void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    private ArrayList<Person> getRecents() {
+        List<Recent> recentList = recentsManager.getAllFromDb();
+        ArrayList<Person> personList = new ArrayList<>(recentList.size());
+        for (Recent r : recentList) {
+            String[] t = r.getName()
+                          .split("\\$");
+            Person p = new Person();
+            p.setId(t[0]);
+            p.setName(t[1]);
```

Comment:
When you're at it:
This should really be extracted to a function `Person.fromRecent()`, so we can maybe use this in a Kotlin `Iterable.map(Person.fromRecent(it))`